src/pages/Inventory.page.js

```diff
@@ -11,12 +11,58 @@ class InventoryPage extends BaseSwagLabPage {
         return $$('.inventory_item');
     }
 
+    get sortDropdown() {
+        return $('select.product_sort_container');
+    }
+
+    getItemName(item) {
+        return item.$('.inventory_item_name');
+    }
+
     get addItemToCartButton() {
         return $$('[id^="add-to-cart"]');
     }
 
     async addItemToCartById(id) {
         await this.addItemToCartButton[id].click();
+}
+
+    async getSortedItems(option) {
+        const inventoryItems = await this.inventoryItems;
+
+        if (!inventoryItems.length) {
+            throw new Error('No inventory items found on the page');
+        }
+
+        const itemNames = [];
+        for (const item of inventoryItems) {
+            const itemName = await this.getItemName(item).getText();
+            itemNames.push(itemName);
+        }
+
+        switch (option) {
+            case 'az':
+                return itemNames.sort();
+            case 'za':
+                return itemNames.sort().reverse();
+            case 'lohi':
+                return itemNames.sort((a, b) => a - b);
+            case 'hilo':
+                return itemNames.sort((a, b) => b - a);
```

Comment:
need to sort by item prices instead of names 
<img width="251" alt="image" src="https://github.com/user-attachments/assets/21b8728d-7a61-4a0f-831f-728b7beca54e">